### ISSUE TITLE
Menu doesn't allow for role and permission assignment

### ISSUE THREAD

**Issue by Backtrennpapier** [Fasse/admidio-statistics#10]
### Description

During the installation of the Statistics plugin for Admidio all steps could be performed as expected. After the installation the next step is to verify the module permissions and visibility of the menu items. The instruction is outlining that this should happen via the Menu administration, see: https://www.admidio.org/dokuwiki/doku.php?id=en:plugins:statistics

> "Log in to Admidio and set the appropriate access rights for the newly added menu items Statistics and Statistics Editor. These are automatically transferred to the plugin pages."

> "The access authorizations can be managed from version 3.0 of the plugin via the role assignment in the menu items. There you can assign any roles to a menu item that can see this menu item. The plugin checks this role mapping and displays the corresponding pages only to members of these roles."

The issue is that the configuration of the module permissions and visibility of menu items doesn't show up like on other menu items. Additionally (but this might be no issue) is that the Statistics module doesn't show up under Settings/Module.

I completely removed the plugin and reinstalled it again, but the permission configuration doesn't show up in the menu. 

### Versions

Admidio version: 4.3.5
Statistics version: 3.4.0

![Screenshot 2024-03-17 at 10 01 01](https://github.com/Fasse/admidio-statistics/assets/22151508/0a752ac5-a688-42c5-ab28-2e20994edcb6)
![Screenshot 2024-03-17 at 10 01 34](https://github.com/Fasse/admidio-statistics/assets/22151508/950f3809-3ddd-482f-8f49-7b11091f6a6c)

**Comment by Fasse** [comment]
@Backtrennpapier Could you please try to install the plugin version 3.4.1 that is released here at GitHub and check if the role assignment is now possible?

**Comment by Backtrennpapier** [comment]
Of course, thanks for looking at this so fast. First I uninstalled the 3.4.0 version via /adm_plugins/statistics/install/install.php , then I deployed the 3.4.1 version and installed it. Now the permission and visibility settings show up as expected. I applied a basic permission and visibility configuration and tested it in a private window, settings seems to be applied as far as I could see and test (not accessible via direct link). Thanks!
